### DISCUSSION TITLE
KERI Implementer's Call 07/30/2024

### DISCUSSION THREAD

**Discussion by m00sey** [WebOfTrust/keri#93] [Meetings]
Reports from:
KERIpy - @m00sey (main), @pfeairheller (back up)
SignifyPy - @pfeairheller , @m00sey (back up)
KERIA - @rodolfomiranda , @2byrds (backup)
Signify-TS - @kentbull , @daniellenskjö (backup)
Cesride - @kentbull , @daidoji (backup)
Kerific - @kordwarshuis
WOT-Terms - @henkvancann
signify-browser-extension - @2byrds , @rodolfomiranda (backup)
polaris-web - @2byrds , @rodolfomiranda (backup)

Please add comments ahead of tomorrow's meeting for any additional topics.

Discussion:
Same updates to counter.
Formatting for KERIA.
Question: https://discord.com/channels/1148629222647148624/1148732741572112486/1267709080030679040
Charles: My second question is if `kli exports` an event stream, what imports that event stream?
   - I don't think so.

Kevin to ping each implementor in the keri PR to ensure we're happy with descriptions.

**Comment by m00sey** [comment]
keripy:


Merged:
- https://github.com/WebOfTrust/keripy/pull/817
- https://github.com/WebOfTrust/keripy/pull/822
- https://github.com/WebOfTrust/keripy/pull/823
- https://github.com/WebOfTrust/keripy/pull/826
- https://github.com/WebOfTrust/keripy/pull/828
- https://github.com/WebOfTrust/keripy/pull/829
- https://github.com/WebOfTrust/keripy/pull/830

Open:
- https://github.com/WebOfTrust/keripy/pull/827

Changes requested:
 - https://github.com/WebOfTrust/keripy/pull/816
 - https://github.com/WebOfTrust/keripy/pull/806

**Comment by m00sey** [comment]
keri:

- https://github.com/WebOfTrust/keri/pull/92

**Comment by 2byrds** [comment]
KERIA:
Open PRs:
- https://github.com/WebOfTrust/keria/pull/268
- https://github.com/WebOfTrust/keria/pull/269
- https://github.com/WebOfTrust/keria/pull/271
- https://github.com/WebOfTrust/keria/pull/272
- https://github.com/WebOfTrust/keria/pull/273
- https://github.com/WebOfTrust/keria/pull/274 
- https://github.com/WebOfTrust/keria/pull/275 
- https://github.com/WebOfTrust/keria/pull/276
- https://github.com/WebOfTrust/keria/pull/277

Signify-TS:
Open PRs:

- https://github.com/WebOfTrust/signify-ts/pull/241 
- https://github.com/WebOfTrust/signify-ts/pull/252 
- https://github.com/WebOfTrust/signify-ts/pull/258 
- https://github.com/WebOfTrust/signify-ts/pull/270 
- https://github.com/WebOfTrust/signify-ts/pull/271

signify-browser-extension
Nothing to report

polaris-web
Nothing to report

**Reply by iFergal** [reply]
Just opened https://github.com/WebOfTrust/keria/pull/278 on KERIA for multi-sig IPEX.

Also have a question on the recipient fields (again) but mostly in the context of IPEX to re-open my Signify-TS IPEX PR.

**Reply by iFergal** [reply]
Also migration issue for SignifyGroupHabs: https://github.com/WebOfTrust/keria/issues/257#issuecomment-2233862784

**Comment by daidoji** [comment]
Nothing to report for cesride.
